7-googleplaystore-reviews-ingestion/code/src/functions/process_playstore_reviews/index.ts

```diff
@@ -0,0 +1,282 @@
+import { client, publicSDK } from '@devrev/typescript-sdk';
+import { AxiosResponse } from 'axios';
+import * as gplay from "google-play-scraper";
+import { StringOutputParser, JsonOutputParser } from "@langchain/core/output_parsers";
+import {
+  ChatPromptTemplate,
+  SystemMessagePromptTemplate,
+  HumanMessagePromptTemplate,
+} from "@langchain/core/prompts";
+import { ChatFireworks } from "@langchain/community/chat_models/fireworks";
+
+export type HTTPResponse = {
+  success: boolean;
+  message: string;
+  data: any;
+};
+
+export const defaultResponse: HTTPResponse = {
+  data: {},
+  message: '',
+  success: false,
+};
+
+export class ApiUtils {
```

Comment:
Maybe we can add this class in a separate helper file as this file is getting a little big?

Reply:
done